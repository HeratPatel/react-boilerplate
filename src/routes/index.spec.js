import React from 'react';
import { shallow } from 'enzyme';
import Router from './index';

describe('<Router />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<Router />);
    expect(wrapper).toMatchSnapshot();
  });
});
