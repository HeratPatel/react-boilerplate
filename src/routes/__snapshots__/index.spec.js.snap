// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router /> Rendering 1`] = `
<BrowserRouter>
  <React.Suspense
    fallback={<Loading />}
  >
    <Switch>
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_ctor": [Function],
            "_result": null,
            "_status": -1,
          }
        }
        exact={true}
        path="/"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_ctor": [Function],
            "_result": null,
            "_status": -1,
          }
        }
        exact={true}
        path="/home"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_ctor": [Function],
            "_result": null,
            "_status": -1,
          }
        }
        exact={true}
        path="/login"
      />
      <RouteAuthentication
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_ctor": [Function],
            "_result": null,
            "_status": -1,
          }
        }
        exact={true}
        isLoggedIn={true}
        path="/profile"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_ctor": [Function],
            "_result": null,
            "_status": -1,
          }
        }
      />
    </Switch>
  </React.Suspense>
</BrowserRouter>
`;
